deploy/crds/crd.yaml

```diff
@@ -19,7 +19,7 @@ spec:
   - name: Desired Version
     type: string
     description: The desired bookkeeper version
-    JSONPath: .status.TargetVersion
+    JSONPath: .spec.version
```

Comment:
Why are we changing this?

Reply:
The desired version is showing as an empty field while doing manual installation. This will fix that.

Reply:
Lets not add this change here unless it is needed for fixing the version map issue.

Reply:
alright